src/Solcore/Pipeline/SolcorePipeline.hs

```diff
@@ -29,93 +35,100 @@ pipeline :: IO ()
 pipeline = do
   startTime <- Time.getCurrentTime
   opts <- argumentsParser
+  result <- compile opts
+  case result of
+    Left err -> do
+      putStrLn err
+      exitWith (ExitFailure 1)
+    Right contracts -> do
+      forM_ (zip [1..] contracts) $ \(i, c) -> do
+        let filename = "output" <> show i <> ".core"
```

Comment:
we would previously overwrite the same output.core file in the multiple contract case. this change does mean that the default output file is `output1.core`.

Reply:
Well done! One question - the CI on Github gives this warning:

```
warning: download buffer is full; consider increasing the 'download-buffer-size' setting
```

do you know what to do with it? Does it come from Nix? Or Github?